src/app/worker/task-processor.service.ts

```
@@ -0,0 +1,232 @@
+    if (!task) {
+      throw new Error(`Task with id ${taskId} not found`);
+    }
+
+    const { customerId, invoice, pdfUrl, emailServiceUrl } = task.payload;
+
+    await this.sendEmail(customerId, invoice, pdfUrl, emailServiceUrl);
+  }
+
+  private async fetchOrdersFromExternalApi(
+    customerId: string,
+    dateFrom?: string,
```

ESLint

[error] 'dateFrom' is defined but never used.

(@typescript-eslint/no-unused-vars)